scripts/aem/README.md

```diff
@@ -0,0 +1,66 @@
+## Script for provisioning a DUT with WIP GRUB and Xen
+
+### Setup on first use
+
+```bash
+./aem.sh init   # can specify --depth=1
+vim arm-cfg.sh  # edit repo URL, branches, device files, list of files to copy
+./aem.sh build
+./aem.sh update # generates helper script
+# put files that will be sent to the DUT in webroot/
+cp somewhere/{grub.cfg,bzImage,initramfs.cpio} webroot
+./aem.sh serve  # starts the server and exists
+```
+
+Example `grub.cfg` is in the current directory ([here](./grub.cfg)).
+`bzImage` and `initramfs.cpio` can be found in <http://boot.3mdeb.com/tb/mb2/>.
```

Comment:
Please add that those are quite minimal images, but full kernel and initramfs from most distributions should also work, as long as they are built for Xen.